auth/internal/services/authentication/keycloak/client/keycloak.go

```diff
@@ -70,12 +71,19 @@ func (c *Client) IsActiveToken(token string) (bool, error) {
 }
 
 func (c *Client) GetAccountIDByJWTToken(token string) (uuid.UUID, error) {
-	userInfo, err := c.GetUserInfo(c.removeBearer(token))
+	accessToken, _, err := new(jwt.Parser).ParseUnverified(c.removeBearer(token), jwt.MapClaims{})
+
 	if err != nil {
-		return uuid.Nil, errors.Wrap(err, keycloakEnums.MessageFailedToGetAccountIDFromKeycloakToken)
+		return uuid.Nil, errors.Wrap(err, keycloakEnums.MessageFailedToParseKeycloakToken)
+	}
+
+	if claims, isValid := accessToken.Claims.(jwt.MapClaims); isValid {
+		if subString, ok := claims["sub"].(string); ok {
```

Comment:
The `ParseWithClaims()` return `key is of invalid type`.
Because it's an external JWT, and we don't have the encryption key, I don't think we'll be able to use this function.

Reply:
sad :(, ok then.